extensions/azurecore/src/extension.ts

```diff
@@ -236,8 +236,9 @@ export async function activate(context: vscode.ExtensionContext): Promise<azurec
 			requestType: azurecore.HttpRequestMethod,
 			requestBody: any,
 			ignoreErrors: boolean,
-			host: string = 'https://management.azure.com'): Promise<azurecore.AzureRestResponse> {
-			return azureResourceUtils.makeHttpRequest(account, subscription, path, requestType, requestBody, ignoreErrors, host);
+			host: string = 'https://management.azure.com',
+			requestHeaders: { [key: string]: string } = {}): Promise<azurecore.AzureRestResponse> {
```

Comment:
Does ADS support different clouds or only public cloud?  if so this api URL would need to be based on current cloud.

Reply:
That's a good catch @brian-harris! Yeah @aasimkhan30 this code should be using the currently configured auth settings to get the correct proper ARM endpoint.